Makefile

```diff
@@ -82,5 +82,9 @@ lint:
 
 fmt:
 	gofmt -s -l -w $(SRCS)
+
+clean:
+	sudo rm -rf dist static site autogen/gen.go traefik
```

Comment:
`./traefik` is created by `go build .`.

There is a structural problem with the Makefile: artefacts built using Docker are owned by root, not by the current user.

It should be possible to `make clean`. When `make clean` is executed, all generated artefacts should be deleted. This is an important requirement for developers. As it stands, it may be necessary to `sudo make clean`, which is an adequate interim compromise. Ideally this problem would be solved more completely.